portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,42 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+table, th, td {
+  border: 1px solid black;
+}
+</style>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Charles Lovett's Portfolio</h1>
+      <p>This Portfolio contians my life growing up and during college. Hope you enjoy learning a little bit about me.</p>
+      <p>Real Quick if you want to know some fun facts about me press the button below:</p>
+      <button onclick="addRandomFunFact()">Fun Facts</button>
       <div id="greeting-container"></div>
+
+    
+    </div>
+    <div id='choice' >
```

Comment:
Please use double quotes for attribute values per the [Google style guide](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks).